### PR TITLE
Add note about environment on WSL

### PR DESCRIPTION
**Pull request by Pilchie**

### CODE REVIEW

**Comment by Pilchie** [comment]
@BrennanConroy - do I remember right that you were struggling with trying to get this working a bit ago?

**Comment by BrennanConroy** [comment]
Nope, not me. Maybe Justin

**Comment by Pilchie** [comment]
@jkotalik?

File: docs/BuildFromSource.md [hunk 1]
```diff
@@ -139,6 +139,11 @@ source activate.sh
 code .
 ```
 
+Note that if you are using the "Remote-WSL" extension in VSCode, the environment is not supplied
+to the process in WSL.  You can workaround this by explicitly setting the environment variables
+in `~/.vscode-server/server-env-setup`.
```
Comment (jkotalik) [inline docs/BuildFromSource.md]: Which environment variables are you referring to? Also, is Remote-WSL the same as WSL2 here?

Comment (Pilchie) [reply docs/BuildFromSource.md]: Env vars are: `DOTNET_ROOT`, `DOTNET_MULTILEVEL_LOOKUP` and `PATH`.

"Remote-WSL" is the name of the VSCode extension.  It works with both WSL1 and WSL2.

Comment (jkotalik) [reply docs/BuildFromSource.md]: Are these environment variables listed elsewhere and what values they should be set to? I don't see them else in this document.

**Review by jkotalik — APPROVED** [review]

**Comment by Pilchie** [comment]
I think the rest of the document tries to use `activate.sh` to abstract over what they are exactly.

**Pull request merged**